src/realm/table.cpp

```diff
@@ -2243,9 +2243,16 @@ ConstTableView Table::find_all_null(ColKey col_key) const
 
 TableView Table::get_distinct_view(ColKey col_key)
 {
-    TableView tv(TableView::DistinctView, m_own_ref, col_key);
-    tv.do_sync();
-    return tv;
+    if (this->has_search_index(col_key)) {
```

Comment:
There's a couple tests you can enable now: `Table_DistinctFloat` and `Table_DistinctDouble`. They are currently commented out because we don't support indexing those types.

Reply:
I am more interested in finding out if `Table::get_distinct_view` could be removed. I don't think it is used anywhere.

Reply:
I believe the only place that was called was to verify PK uniqueness post-migration and now that's handled by core directly.